opennms-doc/guide-install/src/asciidoc/text/monitor-location-minion/docker/minion-docker.adoc

```diff
@@ -0,0 +1,107 @@
+      - OPENNMS_BROKER_USER=minion-user<7>
+      - OPENNMS_BROKER_PASS=minion-password
+      - OPENNMS_HTTP_URL=http://horizon-instance:8980/opennms<8>
+      - OPENNMS_HTTP_USER=minion-user<9>
+      - OPENNMS_HTTP_PASS=minion-password
+    command: ["-f"]
+----
+<1> Friendly container name
+<2> If you process UDP data like SNMP traps, Syslogs or flows, `network_mode: host` ensures the UDP source addresses are not modified
+<3> Time zone for the _Minion_
+<4> A defined identifier for this _Minion_. If not set, a unique user identifier (_UUID_) will be generated
+<5> The name of the location of the _Minion_ and the connection to the _ActiveMQ_ broker running in _{opennms-product-name}_
+<6> _ActiveMQ_ broker endpoint from _{opennms-product-name}_
+<7> Authentication for ActiveMQ broker
+<8> _REST_ endpoint to connect to the _{opennms-product-name}_ instance
+<9> Authentication for the _REST_ endpoint
+
+NOTE: In this example we haven't set credentials to connect the _Minion_ via _REST_ and the _ActiveMQ Message Broker_.
+      The _Minion_ will fall back and uses the default admin/admin credentials for the communication.
+      Permissions for _ActiveMQ_ and _REST_ are assigned with the role _ROLE_MINION_ on the _{opennms-product-name}_ instance.
+
+WARNING: In case you process UDP data and you don't use `network_mode: host` the UDP source address from your packets will be modified from Docker.
+         The source address is your Docker internal gateway instead of the source address of your device.
+         Source addresses are used to associate the Syslog or SNMP traps to the nodes in the OpenNMS database.
+         You can use an isolated network and publish ports as usual if you don't receive UDP based monitoring data.
+         If you don't use `network_mode: host` you have to publish the listener ports manually.
+
+===== Step 2: Start the service stack and test the functionality
+
+[source,shell]
+----
+cd opennms-minion
+docker-compose up -d
+----
+
+===== Step 3: Run _Minion_ health check
+
+.Log in to the _Minion Karaf Shell_ and run the health check
+[source, shell]
+----
+ssh admin@localhost -p 8201
+
+admin@minion> health:check
+Verifying the health of the container
+
+Connecting to OpenNMS ReST API   [ Success  ]
+Verifying installed bundles      [ Success  ]
+Connecting to JMS Broker         [ Success  ]
+
+=> Everything is awesome
+----
+
+NOTE: The default admin password for the _Minion Karaf Shell_ is _admin_.
+
+===== Step 4: Verify status in the administrative Web UI
+
+* Log in as admin in the _{opennms-product-name}_ web interface
+* _Configure OpenNMS -> Manage Minions_. The _Minion_ should be registered and the status should be _up_
+* Verify if _Minion_ is provisioned automatically by going to _Info -> Nodes_ and select the _Minion_. The services _JMX-Minion_, _Minion-Heartbeat_ and _Minion-RPC_ should be _up_ and provisioned on the local loop-back interface
```

Comment:
```suggestion
* Verify that _Minion_ is provisioned automatically by going to _Info -> Nodes_ and selecting the _Minion_. The services _JMX-Minion_, _Minion-Heartbeat_ and _Minion-RPC_ should be _up_ and provisioned on the local loop-back interface
```

Reply:
Could we offer suggestions on what to look for if they are not provisioned? Or is that beyond the scope of this material?